frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -29,6 +36,27 @@ export const hasOCSTaint = (node: NodeKind) => {
   return taints.some((taint: Taint) => _.isEqual(taint, ocsTaint));
 };
 
+export const makeTaintNodesRequest = (selectedNode: NodeKind[]): Promise<NodeKind>[] => {
+  const taintNodesRequest = selectedNode
+    .filter((node: NodeKind) => {
+      const roles = getNodeRoles(node);
+      // don't taint master nodes as its already tainted
+      return roles.indexOf('master') === -1;
+    })
```

Comment:
We need to remove the filter logic as we don't show nodes based on nodeRoles. If the master node is not tainted we can still taint it.